scripts/process-image-manifests

```diff
@@ -52,21 +52,55 @@ extract_manifests_from_image() {
     ls -la ${MANIFEST_TMP_DIR}
 }
 
+set_package_name_for_current_csv() {
+  # In the case of 3scale the package name needs to be customised
+  case $PRODUCT in
+  "3scale")
+    CURRENT_CSV_FILE_NAME=${PRODUCT}-operator.package.yaml
+    ;;
+  *)
+    CURRENT_CSV_FILE_NAME=${PRODUCT}.package.yaml
+  esac
+}
+
```

Comment:
Any reason you are calling this set* and all the rest are get* even though they pretty much all set values?

Would just calling it "get_current_csv_filename" not be better to match the var it's going to set?

Reply:
No real reason. Set made sense to me and I didn't notice the rest were already gets, i'll make that change.

Reply:
Actually this was the commit that @laurafitzgerald added so she should speak to whether there was a reason for it